app/src/main/res/menu/crime_detail_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Naming is much better, but still needs a small change.
See naming convention: https://jeroenmols.com/blog/2016/03/07/resourcenaming/